### PR TITLE
Installation updates

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

File: modules/project-docs/pages/sdk-full-installation.adoc [hunk 1]
```diff
@@ -63,51 +78,57 @@ Then follow the <<post-installation,post-installation steps>>.
...
-|PHP 8.1 VC16 NTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.1-nts-x64-openssl.zip[php_couchbase-3.2.2-8.1-nts-x64-openssl.zip]
-|PHP 8.1 VC16 ZTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.1-zts-x64-openssl.zip[php_couchbase-3.2.2-8.1-zts-x64-openssl.zip]
-|===
-
-Then follow the post-installation steps.
+Currently there are no pre-built binaries for the Windows platform of PHP SDK 4.0.0.
+Instead, you will need to follow https://github.com/couchbase/couchbase-php-client/blob/4.0.0/WINDOWS.md[these steps] to build an extension along with the PHP interpreter.
+This process will be simplified in a future 4.0.x release where the pre-compiled binaries will be published.
+
+// TODO: Add this back when this is available in a future 4.0.x release
```
Comment (maria-robobug) [inline modules/project-docs/pages/sdk-full-installation.adoc:85]: Seeing as we mention this in the release notes, I added this here to ensure users are aware.
I guess we will update this in a future SDK release @avsej?

File: modules/project-docs/pages/sdk-full-installation.adoc [hunk 2]
```diff
@@ -130,20 +151,22 @@ extension=couchbase
 NOTE: on PHP version 7.2.0 and up, simply using the extension name is preferred.
 Previously, you might have used `couchbase.so` or `couchbase.dll`, depending on your platform.
 
-=== PHP Composer
+// TODO: unclear if this is still valid, there is no 4.0.0 package available at https://packagist.org/packages/couchbase/couchbase
```
Comment (maria-robobug) [inline modules/project-docs/pages/sdk-full-installation.adoc]: Unsure if this is still valid, or how we setup composer to work with SDK 4.0.0, please let me know if you have any suggestions @avsej .

File: modules/project-docs/pages/sdk-full-installation.adoc (hunk 1 above)
Comment (avsej) [reply modules/project-docs/pages/sdk-full-installation.adoc:85]: Yes, this is correct

File: modules/project-docs/pages/sdk-full-installation.adoc (hunk 2 above)
Comment (avsej) [reply modules/project-docs/pages/sdk-full-installation.adoc]: the packagist is hosting pure-PHP libraries. Ours is mixing PHP and C++, and we host it on pecl.php.net. The code that was hosted on the packagist was for 3.x branch, and was only implementing datastructures. Now this code  is integrated into 4.0.0, and we don't use packagist anymore.

**Review by avsej — APPROVED** [review]

**Pull request merged**